torch/_prims/__init__.py

```diff
@@ -1814,14 +1814,17 @@ def _as_strided_scatter_meta(
     utils.validate_shape(size)
     utils.validate_strides(stride)
 
-    required_size = utils.compute_required_storage_length(size, stride, storage_offset)
+    input_length = utils.compute_required_storage_length(
+        input.size(), input.stride(), input.storage_offset()
+    )
+    output_length = utils.compute_required_storage_length(size, stride, storage_offset)
```

Comment:
NIT: This quantity doesn't really have anything to do with the output tensor's geometry, so something like `required_length` or `view_length` would be a better fit.

Reply:
Thanks!  that make sense, i'll update the name